pages/content/posts/css-containment.md

```diff
@@ -0,0 +1,54 @@
+---
+author: "Estelle Weyl"
+title: "CSS containment"
+date: "2024-03-19"
+description: "Improving CSS containment documentation"
+tags: ["projects"]
+ShowToc: false
+ShowBreadCrumbs: false
+excerpt: "The Open Web Docs team is improving documentation for all Baseline features. Most recently, the CSS containment module has been revised, along with a new guide on using size and style container queries."
+---
+
+One of the Open Web Docs's 2024 goals is to [document features supported by the three browser engines](https://github.com/openwebdocs/project/issues/153), including ensuring all [Interop features](https://wpt.fyi/results/) are fully documented.
+
+We are starting by documenting all the features of [Interop 2021-22](https://github.com/openwebdocs/project/issues/189) and [2023](https://github.com/openwebdocs/project/issues/190). In Q4, we will focus on the [Interop 2024 features](https://github.com/openwebdocs/project/issues/191) that will be newly supported.
+
+## CSS containment project
+
+The first sub-project completed was [CSS containment](https://github.com/openwebdocs/project/issues/195). CSS containment is part of both [Interop 2022](https://wpt.fyi/interop-2022) and [Interop 2023](https://wpt.fyi/interop-2023), and is still being worked on as part of [Interop 2024](https://wpt.fyi/interop-2024).
+
+Containment improves rendering performance by isolating sections of content for delayed layout and rendering. Container queries, also defined in the CSS containment specification, enable targeting styles based on a container's size or style features.
+
+We created a new [CSS containment module](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Containment) landing page. The module landing page includes a reference with links to all the properties, at-rules and descriptors, CSS functions, events, interfaces defined in the CSS containment specification, and guides. All of these references were reviewed and updated as necessary.
+
+The larger component of this project included updating the existing containment guides and creating new ones.  We also created a new guide.
+
+### CSS containment guides
+
+There are now three CSS containment guides on MDN:
+
+- : [CSS container queries](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_containment/Container_queries)
```

Comment:
Definition lists don't seem to work correctly on the blog.

![image](https://github.com/openwebdocs/owd-website/assets/1152698/13643af9-93e4-42af-9000-3cc01d1372a0)

Maybe use a UL instead?

Reply:
thanks. Went with UL.